### DISCUSSION TITLE
PEST meaning

### DISCUSSION THREAD

**Discussion by AidasK** [pestphp/pest#654] [General]
Hi, just wandering, what does this word means? Maybe:

p - php
e - elegant 
s - simple
t - tests?

Or is it just means Insect - pest? Any history how you can up with it? 

Thanks!

**Comment by owenvoke** [comment]
PHP ~T~est 🤷🏻 Afaik.

**Comment by JonPurvis** [comment]
I believe the story goes that Nuno came up with the idea for Pest, whilst with Freek Van der Herten from Spatie. He wanted to create a PHP Testing Framework like _Jest_, which is a Javascript Testing Framework, so...

PHP + Jest = Pest

Perhaps @nunomaduro can confirm though!

**Reply by phh** [reply]
Pretty sure you're right. Only:

PHP + Test = Pest
